static/usage/v8/radio/label-wrap/demo.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Radio</title>
+    <link rel="stylesheet" href="../../common.css" />
+    <script src="../../common.js"></script>
+    <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@8/dist/ionic/ionic.esm.js"></script>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core@8/css/ionic.bundle.css" />
+
+    <style>
+      ion-list {
+        width: 250px;
+      }
+
+      ion-radio.wrapped::part(label) {
+        text-wrap: wrap;
+      }
+    </style>
+  </head>
+
+  <body>
+    <ion-app>
+      <ion-content>
+        <div class="container">
+          <ion-list>
+            <ion-radio-group value="truncated">
+              <ion-item>
+                <ion-radio value="truncated">Truncated with ellipsis by default</ion-radio>
+              </ion-item>
+              <ion-item>
+                <ion-radio value="wrapped" class="wrapped">`text-wrap` applied to `label` shadow part</ion-radio>
+              </ion-item>
+            </ion-radio-group>
```

Comment:
```suggestion
            <ion-radio-group value="truncated">
              <ion-item>
                <ion-radio value="truncated">
                  Truncated with ellipsis by default
                </ion-radio>
              </ion-item>
              <ion-item>
                <ion-radio value="wrapped-part" class="wrapped">
                  Wrapping with text-wrap applied to label shadow part
                </ion-radio>
              </ion-item>
              <ion-item>
                <ion-radio value="wrapped-div">
                  <div class="ion-text-wrap">
                    Wrapping with ion-text-wrap class applied wrapper element
                  </div>
                </ion-radio>
              </ion-item>    
            </ion-radio-group>      
```

Thank you for this! Do you think we should include an example using a wrapper div with our [ion-text-wrap css utility](https://ionicframework.com/docs/layout/css-utilities#text-modification)?

Reply:
Good call, I did not think to look at the utility classes. I will rework the JS/Angular/React/Vue examples to include the utility class.